src/shogun/kernel/GaussianARDKernel.h

```diff
@@ -4,25 +4,60 @@
  * the Free Software Foundation; either version 3 of the License, or
  * (at your option) any later version.
  *
+ * Written (W) 2015 Wu Lin
  * Written (W) 2012 Jacob Walker
  *
  * Adapted from WeightedDegreeRBFKernel.h
  */
 
-#ifndef GAUSSIANARDKERNEL_H_
-#define GAUSSIANARDKERNEL_H_
+#ifndef _GAUSSIANARDKERNEL_H_
+#define _GAUSSIANARDKERNEL_H_
```

Comment:
There is some standard that say #defines should not start with underscores. 

http://stackoverflow.com/questions/6756059/c-using-c-code-using-double-underscores-in-defines-and-identifiers

not a big thing but maybe good to clean up in the next PR?